targets/targets.json

```diff
@@ -6384,6 +6389,7 @@
             "SPISLAVE",
             "SPI_ASYNCH"
         ],
+        "release_versions": ["2", "5"],
```

Comment:
As this is not yet completed, do we keep release_versions ?

Reply:
I'm not sure how this is supposed to work, @cmonr ?

Reply:
Yup, it's needed to test.

I think only `5` is needed if non-rtos is still not working.

Reply:
@cmonr, Looking through targets.json it seems most targets have both 2 and 5. I think it's required to keep new versions of Mbed 2 working with the SAML21 target. I know at least one company which uses this.